.flake8

```diff
@@ -5,5 +5,7 @@ doctests = False
 max-doc-length = 120
 max-line-length = 100
 statistics = True
+# E203 - whitespace before ':' - PEP8 non-compliant warning in slicing expressions
+extend-ignore = E203
```

Comment:
Please adjust also remaining checks according to the black-onboarding document

Reply:
It didn't trigger on our code-base so I kinda missed it, added.